server/api/after_auth_handler.py

```diff
@@ -20,9 +21,18 @@ def authorize():
         g_user = gauth.google.userinfo()
         db_user = user.User(g_user).save("users")
         session['profile'] = user_info
+        if type(db_user) is dict:
```

Comment:
Which one are we expecting? Does it vary? What changes it? A comment in the code to suggest which path it would take could be handy if it does vary

Reply:
ok. I will write explanatory comment. It looks odd, because if we insert brand new object in database then it return object with one property inserted_id but if we fetching existing object then it return dict.